src/dynatrace/azext_dynatrace/aaz/latest/dynatrace/monitor/_list.py

```diff
@@ -0,0 +1,355 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "dynatrace monitor list",
+)
+class List(AAZCommand):
+    """List all monitor resource by resource group and monitor name
+
+    :example: List monitor
+        az dynatrace monitor list -g rg --monitor-name monitor
```

Comment:
This example seems invalid. It doesn't have `--monitor-name` argument in command